modules/reindex/src/test/resources/responses/main/crazy.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name" : "Crazy Node With Weird Stuff In The Response",
+  "cluster_name" : "distribution_run",
+  "cats": "knock things over",
```

Comment:
adding the cluster_uuid at this level would have done it :) At this point you could also add a crazy object and a crazy array though :)